src/common/components/Modal/AlertModal/AlertModal.tsx

```diff
@@ -0,0 +1,50 @@
+import classNames from "classnames/bind";
+import styles from "@/common/components/Modal/AlertModal/AlertModal.module.scss";
+import { useRef } from "react";
+import Check from "@/images/ic_check.svg";
+import Modal from "@/common/components/Modal/Modal";
+import useOutSideClick from "@/common/components/Modal/hooks/useOutsideClick";
+import Button from "@/common/components/Button/Button";
+
+const cn = classNames.bind(styles);
+
+interface AlertModalProps {
+  message: string;
+  className: string;
+  cancelButtonOnClick: () => void;
+  confirmButtonOnClick: () => void;
```

Comment:
이벤트 핸들러의 컨벤션인 handle, on은 맨 앞에 표현해주면 좋을 것 같습니다~
```suggestion
  onCancelButtonClick: () => void;
  onConfirmButtonClick: () => void;
```